src/flashbar/collapsible-flashbar.tsx

```diff
@@ -369,3 +368,6 @@ const NotificationTypeCount = ({
     </span>
   );
 };
+
+const ListWrapper = ({ children, withMotion }: { children: ReactNode; withMotion: boolean }) =>
+  withMotion ? <TransitionGroup component={null}>{children}</TransitionGroup> : <>{children}</>;
```

Comment:
Added this small component because TransitionGroup adds an extra div by default unless `component` is null, but using this same prop in React.Fragment triggers runtime (not build time) errors.

An alternative would be to have this inline inside the main component:
```
const ListWrapper = isReducedMotion ? TransitionGroup : React.Fragment;
(...)
<ListWrapper {...isReducedMotion && { component: null } }>
  (...)
</ListWrapper>
```
Equally valid, just a bit uglier IMO.

Reply:
Fine by me